docs/content/tutorial/advanced-tutorial/pipelines.mdx

```diff
@@ -243,7 +243,7 @@ Each of the ways we can invoke a Dagster pipeline lets us select which mode we'd
 From the command line, we can set `-d` or `--mode` and select the name of the mode:
```

Comment:
Could you also update this line to remove the `-d` flag

Reply:
You bet! I just updated and pushed the change. This is my first time to do PRs on a large project like this - I appreciate your help.